<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eml</class>
 <widget class="QWidget" name="eml">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>660</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>send</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>720</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>a.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>461</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background-color: rgb(0,171, 159);
               background-image:-moz-linear-gradient(top, #ff3443, #ad232e);
               background-image:-o-linear-gradient(top, #ff3443, #ad232e);
               background-image:-webkit-gradient(linear, center top, center bottom,
                    from(#ff3443), to(#ad232e));
               background-image:linear-gradient(top, #ff3443, #ad232e);
               width:90px;
border-radius: 12px;
}
QLineEdit:hover {
  background: #0eb082;

  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
   </property>
   <property name="text">
    <string extracomment="TO"/>
   </property>
   <property name="cursorPosition">
    <number>0</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>290</y>
     <width>461</width>
     <height>321</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit
{
  background: #0eb082;
   
               background-image:-moz-linear-gradient(top, #ff3443, #ad232e);
               background-image:-o-linear-gradient(top, #ff3443, #ad232e);
               background-image:-webkit-gradient(linear, center top, center bottom,
                    from(#ff3443), to(#ad232e));
               background-image:linear-gradient(top, #ff3443, #ad232e);
               width:90px;
border-radius: 12px;
}
QTextEdit:hover {

 background-color: rgb(0,171, 159);
  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>250</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit
{
    background-color: rgb(0,171, 159);
               background-image:-moz-linear-gradient(top, #ff3443, #ad232e);
               background-image:-o-linear-gradient(top, #ff3443, #ad232e);
               background-image:-webkit-gradient(linear, center top, center bottom,
                    from(#ff3443), to(#ad232e));
               background-image:linear-gradient(top, #ff3443, #ad232e);
               width:90px;
border-radius: 12px;
}
QLineEdit:hover {
  background: #0eb082;

  box-shadow: 0 5px #666;
  transform: translateY(4px);
}</string>
   </property>
   <property name="text">
    <string>object:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>660</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>return</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TO:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_33">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Agenda</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_35">
   <property name="geometry">
    <rect>
     <x>980</x>
     <y>10</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_21">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Statistics</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_32">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Products</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_34">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>clients</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>480</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Reclamations</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_30">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Employees</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>lineEdit</zorder>
  <zorder>textEdit</zorder>
  <zorder>lineEdit_2</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>label_2</zorder>
  <zorder>pushButton_33</zorder>
  <zorder>pushButton_35</zorder>
  <zorder>pushButton_21</zorder>
  <zorder>pushButton_32</zorder>
  <zorder>pushButton_34</zorder>
  <zorder>pushButton_24</zorder>
  <zorder>pushButton_30</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
